smoke/internal/testdata/db/admin.json

```diff
@@ -1,4 +1,4 @@
-{ "_id" : "service_flags", "github_status_api_disabled" : false, "alerts_disabled" : false, "repotracker_disabled" : false, "scheduler_disabled" : false, "check_blocked_tasks_disabled": false, "github_pr_testing_disabled" : false, "repotracker_push_event_disabled" : false, "cli_updates_disabled" : false, "task_dispatch_disabled" : false, "s3_binary_downloads_disabled": true, "large_parser_projects_disabled": true, "monitor_disabled" : false, "notifications_disabled" : false, "taskrunner_disabled" : false, "background_stats_disabled" : true, "event_processing_disabled" : false, "webhook_notifications_disabled" : false, "jira_notifications_disabled" : false, "slack_notifications_disabled" : false, "email_notifications_disabled" : false, "task_logging_disabled" : false, "cache_stats_job_disabled" : false, "evergreen_test_results_disabled": false, "agent_start_disabled" : false, "host_init_disabled" : false, "pod_init_disabled": true, "cache_stats_endpoint_disabled" : false, "host_allocator_disabled" : false, "pod_allocator_disabled": false, "task_reliability_disabled" : false, "background_reauth_disabled": true, "background_cleanup_disabled": false, "cloud_cleanup_disabled": true, "unrecognized_pod_cleanup_disabled": true, "sleep_schedule_disabled": true, "static_api_keys_disabled": true}
+{ "_id" : "service_flags", "github_status_api_disabled" : false, "alerts_disabled" : false, "repotracker_disabled" : false, "scheduler_disabled" : false, "check_blocked_tasks_disabled": false, "github_pr_testing_disabled" : false, "repotracker_push_event_disabled" : false, "cli_updates_disabled" : false, "task_dispatch_disabled" : false, "s3_binary_downloads_disabled": true, "large_parser_projects_disabled": true, "monitor_disabled" : false, "notifications_disabled" : false, "taskrunner_disabled" : false, "background_stats_disabled" : true, "event_processing_disabled" : false, "webhook_notifications_disabled" : false, "jira_notifications_disabled" : false, "slack_notifications_disabled" : false, "email_notifications_disabled" : false, "task_logging_disabled" : false, "cache_stats_job_disabled" : false, "evergreen_test_results_disabled": false, "agent_start_disabled" : false, "host_init_disabled" : false, "pod_init_disabled": true, "cache_stats_endpoint_disabled" : false, "host_allocator_disabled" : false, "pod_allocator_disabled": false, "task_reliability_disabled" : false, "background_reauth_disabled": true, "background_cleanup_disabled": false, "cloud_cleanup_disabled": true, "unrecognized_pod_cleanup_disabled": true, "sleep_schedule_disabled": true, "static_api_keys_disabled": true, "use_jwt_token_for_cli": true}
```

Comment:
This seems like the JWT token should be disabled for the smoke test? Unless you plan to add the JWT to the smoke test.

Reply:
this shouldn't impact the smoke test because it should run as a service user. If it doesn't yet and breaks when I start using the flag, I'll update it.